pkg/gc/components.go

```diff
@@ -0,0 +1,20 @@
+package gc
+
+import (
+	"time"
+
+	config_core "github.com/kumahq/kuma/pkg/config/core"
+	"github.com/kumahq/kuma/pkg/core"
+	"github.com/kumahq/kuma/pkg/core/runtime"
+)
+
+var (
+	gcLog = core.Log.WithName("garbage-collector")
+)
+
+func Setup(rt runtime.Runtime) error {
+	if rt.Config().Environment != config_core.UniversalEnvironment {
```

Comment:
explain please why this is run only on Universal, something like:
```
Dataplane GC is run only on Universal because on Kubernetes Dataplanes are bound by ownership to Pods. Therefore on K8S offline dataplanes are cleaned up quickly enough to not run this.
```
feel free to rephrase